### DISCUSSION TITLE
I keep getting an error when I try to follow the instructions in the readme

### DISCUSSION THREAD

**Discussion by vcnwafor** [suyuan32/simple-admin-core#26] [Q&A] [answered]
`go run core.go -f etc/core.yaml`

2022/10/24 03:08:16 mkdir /home/data: operation not supported
exit status 1

**Comment by suyuan32** [comment]
In minikube you should create the directory by yourself, and change the permission to 777. You can just enter the container to do this. If you run the code in your local machine, you can change the storage path or make the directory by yourself like minikube.

**Comment by suyuan32** [comment]
You got the errors due to permisson.

**Comment by suyuan32** [comment, marked as answer]
modify the etc/core.yaml

`Log:
  ServiceName: coreRpcLogger
  Mode: file
  Path: /home/data/logs/core/rpc
  Encoding: json
  Level: info
  Compress: false
  KeepDays: 7
  StackCoolDownMillis: 100`

change the log path

**Comment by suyuan32** [comment]
https://suyuan32.github.io/simple-admin-core/#/simple-admin/en/docs/env_setting

**Reply by vcnwafor** [reply]
Thanks
